File: ClasePersona/ClasePersona/Persona.vb
Public Class Persona
    'Seccion de atributos
    Private nombre As String
    Private apellido As String
    Private edad As Integer

    'Seccion de constructores
    'Constructor sin parametros

    Sub New()
    End Sub

    Sub New(ByVal nombre1 As String, ByVal apellido1 As String, ByVal edad1 As Integer)
        Me.nombre = nombre1
        Me.nombre = apellido1
        Me.edad = edad1
    End Sub

    'Sets()
    Sub setNombre(ByVal nombre1 As String)
        Me.nombre = nombre1
    End Sub

    Sub setApellido(ByVal apellido1 As String)
        Me.apellido = apellido1
    End Sub

    Sub setEdad(ByVal edad1 As Integer)
        Me.edad = edad1
    End Sub

    'Gets()
    Function getNombre() As String
        Return Me.nombre
    End Function

    Function getApellido() As String
        Return Me.apellido
    End Function

    Function getEdad() As Integer
        Return Me.edad
    End Function

    Sub Imprimir()
        Console.WriteLine("Nombre: " + Me.nombre)
        Console.WriteLine("Apellido: " + Me.apellido)
        Console.WriteLine("Edad: " + CStr(Me.edad))
    End Sub

End Class

File: Lista_CSharp/Lista_CSharp/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lista_CSharp
{
    class Persona
    {
        private string nombre;
        private string apellido;
        private int edad;
        private string telefono;

        public Persona()
        {
            nombre = "";
            apellido = "";
            edad = 0;
            telefono = "";
        }

        public void setNombre(string val)
        {
            nombre = val;
        }

        public string getNombre()
        {
            return nombre;
        }
        public void setApellido(string val)
        {
            apellido = val;
        }

        public string getApellido()
        {
            return apellido;
        }
        public void setTelefono(string val)
        {
            telefono = val;
        }

        public string getTelefono()
        {
            return telefono;
        }
        public void setEdad(int val)
        {
            edad = val;
        }

        public int getEdad()
        {
            return edad;
        }
    }
}
